src/vulkan-renderer/application.cpp

```diff
@@ -470,7 +470,8 @@ Application::Application(int argc, char **argv) {
         enable_debug_marker_device_extension = false;
     }
 
-    m_device = std::make_unique<wrapper::Device>(m_instance->instance(), m_surface->get(),
+    // Here we are passing VK_API_VERSION_1_0, as we do not require advanced memory features.
+    m_device = std::make_unique<wrapper::Device>(m_instance->instance(), VK_API_VERSION_1_0, m_surface->get(),
```

Comment:
Maybe it would be a good idea to store the API version in the instance class so they don't become out of sync, like they have here - the instance is being created with `VK_API_VERSION_1_1` but here you are passing `VK_API_VERSION_1_0`. Could also change the `Device` constructor to take in a `const Instance &`.

Reply:
Interestingly I just realize it crashes when I pass `VK_API_VERSION_1_1` to the device.

Reply:
This seems to be some driver issue on my side.